nvflare/app_opt/confidential_computing/snp_authorizer.py

```diff
@@ -29,14 +32,18 @@ class SNPAuthorizer(CCAuthorizer):
     def __init__(self):
         super().__init__()
         self.logger = logging.getLogger(self.__class__.__name__)
+        self.my_nonce_history = NonceHistory(100)
+        self.seen_nonce_history = NonceHistory(5000)
```

Comment:
could we either introduce constants for these default numbers or make them part of the init args of this class?

Reply:
Both will be specified by one argument in the `__init__` now.